src/main/entity/Ticket.java

```diff
@@ -0,0 +1,100 @@
+package main.entity;
+
+import main.entity.abstracts.AbstractTicket;
+import main.util.NullableWarning;
+
+import java.lang.reflect.Field;
+import java.util.Objects;
+
+public class Ticket extends AbstractTicket {
+
+  @NullableWarning
+  private String concertHall;
+  private int eventCode;
+  private long time;
+  private boolean isPromo;
+  private char stadiumSector;
+  private double maxBackpackWeight;
+  private double price;
+  private long createdAt;
+
+
+
+  public Ticket(int ticketID, String hall, int code, long eventTime, boolean promoStatus,
+                char sector, double backpackWeight, double ticketPrice) {
+    setId(ticketID);
+    concertHall = hall;
+    eventCode = code;
+    time = eventTime;
+    isPromo = promoStatus;
+    stadiumSector = sector;
+    maxBackpackWeight = backpackWeight;
+    price = ticketPrice;
+    createdAt = System.currentTimeMillis();
+    checkNullableFields();
+  }
+
+  public Ticket(String hall, int code, long eventTime) {
+    concertHall = hall;
+    eventCode = code;
+    time = eventTime;
+    createdAt = System.currentTimeMillis();
+    isPromo = false;
+    stadiumSector = 'A';
+    maxBackpackWeight = 0.0;
+    price = 0.0;
+    checkNullableFields();
+  }
+
+
+  private void checkNullableFields() {
```

Comment:
extract this method to separate class, it looks like more general method and doesn't belong to Ticket itself